launcher/src/backend/TekuGasLimitConfig.js

```diff
@@ -0,0 +1,23 @@
+export class TekuGasLimitConfig{
+  constructor(nodeConnection) {
+    this.nodeConnection = nodeConnection;
+  }
+
+  async createGasConfigFile(gasLimit, feeRecipient, configPath){
+    await this.nodeConnection.sshService.exec(`cat > ${configPath}/gas_config.json`);
+    await this.nodeConnection.sshService.exec(`echo '{\n\t"default_config":\n\t{\n\t\t"fee_recipient":"${feeRecipient}",\n\t\t"builder":\n\t\t{\n\t\t\t"enabled":true,\n\t\t\t"gas_limit":"${gasLimit}"\n\t\t}\n\t}\n}' > ${configPath}/gas_config.json`);
+  }
+
+  async removeGasConfigFile(configPath){
+    await this.nodeConnection.sshService.exec(`rm -f ${configPath}/gas_config.json`);
+  }
+
+  async readGasConfigFile(configPath){
```

Comment:
imho would be useful to improve error handling: -> e.g: If the gas_limit is not found or the config file doesn't exist